Investigation/Java/Java-auth/auth_server/README.md

```diff
@@ -0,0 +1,105 @@
+# Authentication server
+This is the authentication server which use keycloak as the server implementation
+
+## How to run
+https://www.keycloak.org/getting-started/getting-started-docker
+
+`docker run -p 8080:8080 -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin quay.io/keycloak/keycloak:17.0.1 start-dev`
+
+## How to setup the keycloak server using Api-call (Client credential grant type)
+1. create a realme setting called: automl_dev
+    - <img src="./img/1.jpg" width="60%"/>
+2. import the `realm-export.json`
+3. try: `GET http://localhost:8080/realms/automl_dev/.well-known/openid-configuration`
+    - ```json
+      {
+          ...,
+          "token_endpoint": "http://localhost:8080/realms/automl_dev/protocol/  openid-connect/token",
+          ...
+      }
+      ```
+4. try: `POST http://localhost:8080/realms/automl_dev/protocol/openid-connect/token`
+    - with:  
+        <img src="./img/2.jpg" width="60%"/>  
```

Comment:
img 2 can't display

Reply:
resolved in [1a59426](https://github.com/ml-for-good/automl/pull/52/commits/1a594260fb18cbeba342dbd1a2faeccfc9651de7)